egov-indexer/pqm-service-indexer.yml

```
@@ -17,14 +17,14 @@ ServiceMaps:
               - inJsonPath: $
                 outJsonPath: $.Data.tests
             externalUriMapping:
-              - path: http://egov-workflow-v2.pucar-prod:8080/egov-workflow-v2/egov-wf/process/_search
+              - path: http://egov-workflow-v2.egov:8080/egov-workflow-v2/egov-wf/process/_search
                 queryParam: businessIds=$.testId,history=true,tenantId=$.tenantId
                 apiRequest: { "RequestInfo": { "apiId": "org.egov.pt","ver": "1.0","ts": 1502890899493,"action": "asd","did": "4354648646","key": "xyz","msgId": "654654","requesterId": "61","authToken": "d9994555-7656-4a67-ab3a-a952a0d4dfc8","userInfo": { "id": 1,"uuid": "1fec8102-0e02-4d0a-b283-cd80d5dab067","type": "EMPLOYEE","tenantId": "pg.citya","roles": [ { "name": "Employee","code": "EMPLOYEE","tenantId": "pg.citya" } ] } } }
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)